test/e2e/placement_selecting_resources_test.go

```diff
@@ -856,7 +855,7 @@ var _ = Describe("validating CRP when failed to apply resources", Ordered, func(
 
 var _ = Describe("validating CRP when placing cluster scope resource (other than namespace)", Ordered, func() {
 	crpName := fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())
-	clusterRoleName := fmt.Sprintf("reader-%d", GinkgoParallelProcess())
+	clusterRoleName := fmt.Sprintf("test-cluserrole-%d", GinkgoParallelProcess())
```

Comment:
will rename help to reduce the flakiness?

Reply:
Will check again, but without the rename it was failing locally on my machine.